apps/frontend/src/components/views/PurchaseCredits/steps/Step1_SelectPackage.tsx

```diff
@@ -0,0 +1,190 @@
+                    </div>
+                  </>
+                )}
+
+                {p.features && (
+                  <div className='mt-2 flex flex-col gap-2 text-sm'>
+                    {p.features.map((f) => (
+                      <div
+                        key={f}
+                        className='flex items-center gap-2 text-muted-foreground'
+                      >
+                        <CheckIcon />
+                        <span>{f}</span>
+                      </div>
+                    ))}
+                  </div>
+                )}
+
+                <div className='mt-auto'>
+                  <Button
+                    variant={p.popular ? 'primary' : 'secondary'}
+                    className={cn(
+                      'w-full transition-all duration-300',
+                      p.popular
+                        ? 'hover:bg-primary-hover hover:text-primary-foreground'
+                        : 'hover:bg-secondary-hover hover:text-secondary-foreground',
+                    )}
+                  >
+                    {p.buttonLabel ?? 'Select Package'}
+                  </Button>
+                </div>
+              </div>
+            </Card>
+          ))}
+        </div>
+
+        <div className='mt-8 rounded-xl border bg-muted/30 p-6'>
+          <div className='mb-4 text-lg font-semibold'>How it works</div>
+          <div className='grid grid-cols-1 gap-8 md:grid-cols-3'>
+            <div className='flex items-start gap-3'>
+              <div className='flex h-8 w-8 items-center justify-center rounded-full bg-primary text-sm font-bold text-primary-foreground'>
+                1
+              </div>
+              <div>
+                <div className='font-medium'>Select Package</div>
+                <div className='text-sm text-muted-foreground'>
+                  Choose your storage amount or configure a custom package
+                </div>
+              </div>
+            </div>
+            <div className='flex items-start gap-3'>
+              <div className='flex h-8 w-8 items-center justify-center rounded-full bg-primary text-sm font-bold text-primary-foreground'>
+                2
+              </div>
+              <div>
+                <div className='font-medium'>Connect Wallet</div>
+                <div className='text-sm text-muted-foreground'>
+                  Approve AI3 token transfer from your crypto wallet
+                </div>
+              </div>
+            </div>
+            <div className='flex items-start gap-3'>
+              <div className='flex h-8 w-8 items-center justify-center rounded-full bg-primary text-sm font-bold text-primary-foreground'>
+                3
+              </div>
+              <div>
+                <div className='font-medium'>Instant Credits</div>
+                <div className='text-sm text-muted-foreground'>
+                  Once transaction is confirmed (12 confirmations) you will
```

Comment:
Should this be a constant rather than a hard-coded number of confirmations?

I don't think it matter as much either way, because in normal operation we would only have a 12 block reorg after thousands of years. (However, if there is a network split or high load, it could happen much sooner.)

Reply:
What would you recommend? Should we have a higher confirmation threshold?